src/nanovg_mtl.m

```diff
@@ -1082,6 +1079,8 @@ static int mtlnvg__renderCreate(void* uptr) {
   [backFaceStencilDescriptor release];
   [stencilDescriptor release];
 
+  mtl->clearBufferOnFlush = NO;
```

Comment:
The initial buffer should be cleared with `mtl->clearColor = MTLClearColorMake(0, 0, 0, 0)`.

Reply:
you mean:
```c
mtl->clearBufferOnFlush = YES;
mtl->clearColor = MTLClearColorMake(0, 0, 0, 0);
```
?

Reply:
Never mind. After thinking about it again. The current implementation is fine. 👍